src/core/Link/BaseLink/BaseLink.baseStyles.ts

```diff
@@ -41,5 +41,23 @@ export const baseStyles = (theme: SuomifiTheme) => css`
         text-decoration: none;
       }
     }
+    &.fi-link--accent {
+      text-decoration: none;
+      & .fi-link--accent_icon {
+        font-size: 16px;
```

Comment:
Font size should be rem instead of pixels in --accent & --small

Reply:
I'd keep the size as it is for the icon, since it's always 16px in designs regardless of font size.